DotNet/parser.js

```diff
@@ -10,22 +10,30 @@
 	}
 	let element = ''
 	let options = ''
+	let scrollImplemented = false
 	if (cs.frames === undefined && cs.region === undefined) element = '.Window()'
 	else {
-		if (cs.frames) element += frames(cs.frames)
+		if (cs.frames) {
+			if (cs.scrollRootElement) {
+				element += `.Window()` + scrollRootElement(cs.scrollRootElement)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
				element += ".Window()" + scrollRootElement(cs.scrollRootElement)
```